test/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+require('mocha-eslint')([
+    'lib',
+]);
```

Comment:
This will not validate the `test` directory on normal test runs. Usually we don't care too much about the tests being linted, but since you've already done the work to make the tests valid - why not lint them on every test run. 

Just addin `test` to this list will solve ir